WcaOnRails/spec/factories/competitions.rb

```diff
@@ -34,6 +34,12 @@
       registration_close 2.weeks.from_now.change(usec: 0)
     end
 
+    trait :confirmed do
```

Comment:
Hmm... this isn't what I would have expected from <https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md>. Does this work instead?

```ruby
    trait :confirmed do
      with_delegate
      isConfirmed true
    end
```